src/Microsoft.Framework.PackageManager/Publish/PublishProject.cs

```diff
@@ -419,7 +423,11 @@ private bool UpdateLockFile(PublishRoot root)
                 restoreCommand.CheckHashFile = false;
                 restoreCommand.RestoreDirectories.Add(restoreDirectory);
                 restoreCommand.FeedOptions = feedOptions;
-                restoreCommand.Reports = root.Reports;
+                restoreCommand.Reports = root.Reports.ShallowCopy();
+
+                // Mute "dnu restore" completely if it is invoked by "dnu publish --quiet"
+                restoreCommand.Reports.Information = root.Reports.Quiet;
```

Comment:
Without this, you'll see
![publishmessage_extrarestore](https://cloud.githubusercontent.com/assets/1383883/7760861/746c60a4-ffd4-11e4-86b1-291646e9dc1b.PNG)